services/datamanager/builtin/capture_test.go

```diff
@@ -75,58 +78,67 @@ func TestDataCaptureEnabled(t *testing.T) {
 		initialCollectorDisableStatus bool
 		newCollectorDisableStatus     bool
 		remoteCollector               bool
+		emptyTabular                  bool
 	}{
 		{
-			name:                          "Config with data capture service disabled should capture nothing.",
+			name:                          "Config with data capture service disabled should capture nothing",
 			initialServiceDisableStatus:   true,
 			newServiceDisableStatus:       true,
 			initialCollectorDisableStatus: true,
 			newCollectorDisableStatus:     true,
 		},
 		{
-			name:                          "Config with data capture service enabled and a configured collector should capture data.",
+			name:                          "Config with data capture service enabled and a configured collector should capture data",
 			initialServiceDisableStatus:   false,
 			newServiceDisableStatus:       false,
 			initialCollectorDisableStatus: false,
 			newCollectorDisableStatus:     false,
 		},
 		{
-			name:                          "Disabling data capture service should cause all data capture to stop.",
+			name:                          "Config with data capture service implicitly enabled and a configured collector should capture data",
```

Comment:
Am I understanding correctly that this is a behavior change whereby if capture is configured on a component we automatically set up the dm service/capture?

If so, definitely a welcome change. Just wanted to confirm because the UI shows a warning if somebody configures capture on a component without the service enabled, so I can open a ticket to remove that

Reply:
data capture is already being created implicitly. this PR did not introduce that. it's considered a default service

Reply:
If capture is enabled on a component currently but the service defined in the config, it does not capture anything. Just confirmed using a robot in app + locally using this test/config on `main`. This is because `getServiceConfig` was used to check if there was a dmsvc in the config, and Update would noop if there was not

Reply:
Though if we are making this change I think we might want to revisit syncing being enabled by default - otherwise a user who just configured capture through the component config but did not explicitly configure the data manager service would also automatically have that data synced, which seems like it would be unexpected behavior from the user's perspective

Reply:
Resolved in email thread - we are not changing the default sync disabled value.